tools/README.md

```diff
@@ -1 +1,20 @@
-Command line utilities for processing and converting 3D Tiles tilesets.
\ No newline at end of file
+# 3D Tiles Tools
+
+Command line utilities for processing and converting 3D Tiles tilesets.
+
+## Instructions
+
+Clone this repo and install [Node.js](http://nodejs.org/).  From the root directory of this repo, run:
+```
+npm install
+```
+
+## Tools
+
+### gzipTileset
+
+Detects whether the tileset is compressed or not and does the opposite.
```

Comment:
> I thought we were going to have one processing pipeline and be able to build a series of stages in exactly the same way as we plan to do so for gltf-pipeline.

Ah yeah, I wasn't really thinking about that when making this script but I'll start to support a pipeline approach. gltf-pipeline is about half-way there in terms of the grand vision.

> Why not have separate zip / unzip arguments instead of "doing the opposite." Did you look at other command-line zip tools?

I can do that.

> Is this precise? I would expect it to test each tile so, for example, if 3 of 10 tests are already zipped, it only zips the other 7.

I didn't really expect this to happen in practice, but it shouldn't be hard to support.